plugins/basic/README.md

```diff
@@ -0,0 +1 @@
+every plugin should have a readme file with info.
```

Comment:
Probably the same build instructions for all the plug-ins, to be done as part of `tk-core/developer/build_plugin.py` doc.